client/repolist.c

```diff
@@ -648,7 +648,8 @@ TDNFRepoListFinalize(
        for the repos that are enabled. */
     for(pRepo = pTdnf->pRepos; pRepo; pRepo = pRepo->pNext)
     {
-        if(pRepo->nEnabled)
+//        if(pRepo->nEnabled)
+        if (1)
```

Comment:
???why

Reply:
Oops, I forgot to clean that up.

We need to loop through all repos, even disabled ones, to expand the URLs which we need to determine the cache path.